thirdparty/googletest/BUILD.bazel

```diff
@@ -3,8 +3,16 @@ load("//shared/bazel/rules:packaging.bzl", "package_static_cc_project")
 
 wpilib_cc_library(
     name = "googletest",
-    srcs = glob(["src/**"]),
-    hdrs = glob(["include/**"]),
+    srcs = glob([
+        "src/**/*.cc",
+        "src/**/*.cpp",
+        "src/**/*.h",
+        "src/**/*.hpp",
+    ], allow_empty = True),
+    hdrs = glob([
+        "include/**/*.h",
+        "include/**/*.hpp",
+    ], allow_empty = True),
```

Comment:
Why are we allowing empty glob expansions here?

Reply:
This also shows up in `cscore/BUILD.bazel` and `shared/bazel/rules/cc_rules.bzl`.